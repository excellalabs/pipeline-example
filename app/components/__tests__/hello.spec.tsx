
import * as React from 'react'
import * as renderer from 'react-test-renderer'
import Hello from '../hello'

describe('snapshots', () => {
  it('should render the correct name of "world"', () => {
    const tree = (
      <Hello name='world' />
    )
    expect(tree).toMatchSnapshot()
  })
  it('should render the correct name of "james"', () => {
    const tree = (
      <Hello name='james' />
    )
    expect(tree).toMatchSnapshot()
  })
})
