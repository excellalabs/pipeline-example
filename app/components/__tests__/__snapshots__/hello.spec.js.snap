exports[`snapshots should render the correct name of "james" 1`] = `
<Unknown
  name="james" />
`;

exports[`snapshots should render the correct name of "world" 1`] = `
<Unknown
  name="world" />
`;
